### PR TITLE
CSHARP-816 Set legacy settings from the default profile

### PR DESCRIPTION
**Pull request by joao-r-reis**

### CODE REVIEW

**Comment by joao-r-reis** [comment]
Jenkins builds are failing due to saxon download. I will rerun the build once #480 is merged which fixes this.

File: build.yaml [hunk 1]
```diff
@@ -92,7 +92,7 @@ build:
 
       # Download and uncompress saxon
       mkdir saxon
-      curl -L -o saxon/saxon9he.jar http://central.maven.org/maven2/net/sf/saxon/Saxon-HE/9.8.0-12/Saxon-HE-9.8.0-12.jar
+      curl -L -o saxon/saxon9he.jar https://repo1.maven.org/maven2/net/sf/saxon/Saxon-HE/9.8.0-12/Saxon-HE-9.8.0-12.jar
```
Comment (jorgebay) [inline build.yaml]: In the future we should add saxon to the C# driver Jenkins image.

File: src/Cassandra/Builder.cs [hunk 2]
```diff
@@ -217,7 +214,98 @@ public Configuration GetConfiguration()
...
+                    lbp,
+                    _reconnectionPolicy,
+                    rep,
+                    sep,
+                    _timestampGenerator);
+            }
+
+            if (profile.LoadBalancingPolicy != null && _loadBalancingPolicy != null)
+            {
+                Builder.Logger.Warning(
```
Comment (jorgebay) [inline src/Cassandra/Builder.cs]: Good idea to log a warning in these cases.

File: build.yaml (hunk 1 above)
Comment (joao-r-reis) [reply build.yaml]: good idea, https://datastax-oss.atlassian.net/browse/CSHARP-848

**Pull request merged**